apps/ledger-live-desktop/src/newArch/features/ModularDrawer/components/Search/useSearch.ts

```diff
@@ -0,0 +1,56 @@
+import React, { useMemo, useState } from "react";
+import Fuse from "fuse.js";
+import { getEnv } from "@ledgerhq/live-env";
+import { track } from "~/renderer/analytics/segment";
+import { CryptoOrTokenCurrency } from "@ledgerhq/types-cryptoassets";
+
+export type SearchProps = {
+  setItemsToDisplay: React.Dispatch<React.SetStateAction<CryptoOrTokenCurrency[]>>;
+  source: string;
+  flow: string;
+  items: CryptoOrTokenCurrency[];
+};
+
+export type SearchResult = {
+  handleSearch: (query: string) => void;
+  setSearchQuery?: React.Dispatch<React.SetStateAction<string>>;
```

Comment:
Is it necessary to expose `setSearchQuery` ? It looks to me that the only correct way to update the `searchQuery` is by calling `handleSearch`. Am I missing something ?